src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const categories = (state = [], action) => {
+  switch (action.type) {
+    case 'Under construction':
+      return false;
+
+    default:
+      return state;
+  }
+};
+
+export const checkStatus = () => ({
+  type: 'Under construction',
```

Comment:
- [ ] Kindly define action_type for check status which follow the duck pattern and call it here.